.github/workflows/basic.yml

```diff
@@ -47,10 +53,16 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: 1.65.0
+          toolchain: 1.71.1
           override: true
           components: rustfmt, clippy
 
+      - name: Install protoc
+        uses: arduino/setup-protoc@v1
+        with:
+          version: '3.x'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
Not related to this PR, but I just ran `cargo clippy -- -D warnings` on main of the repo and it fails. Can you check if the clippy check below is even working?